InstallSpinnaker.sh

```diff
@@ -60,7 +60,8 @@ usage: $0 [--cloud_provider <aws|google|none|both>]
 
     If run with no arguments you will be prompted for cloud provider and region
 
-    --cloud_provider <arg>      currently supported are google, amazon and none
+    --cloud_provider <arg>      currently supported are google, azure, amazon and none
```

Comment:
is spacing off here?
dont document both. mark it deprecated below where it is handled.